hands-on-exercise/4-run-in-prod/1-run-prod-docker.md

```diff
@@ -2057,33 +2102,22 @@ You now have a container running both the checkers and a faucet. You are ready t
 
 ### Test your standalone checkers
 
-Prepare the `client/.env` values depending on whether you will run `npm test` locally or in Docker:
-
-<CodeGroup>
-
-<CodeGroupItem title="Local">
-
-```ini
-RPC_URL="http://localhost:26657"
-FAUCET_URL="http://localhost:4500"
-```
-
-</CodeGroupItem>
-
-<CodeGroupItem title="Docker">
+Which `RPC_URL` and which `FAUCET_URL` will the tests require?
 
-```ini
-RPC_URL="http://checkers:26657"
-FAUCET_URL="http://cosmos-faucet:4500"
-```
+* If locally, the defaults will do:
 
-Where you insert the `--name`s used when launching the Docker containers.
+    ```ini
+    RPC_URL="http://localhost:26657"
+    FAUCET_URL="http://localhost:4500"
+    ```
 
-</CodeGroupItem>
+* If running from Docker, you have to pass to the tests values that resolve within Docker's automatic name resolution within `checkers-net`:
```

Comment:
@xavierlepretre Can we switch "within" in either of these cases? Such as "..._within_ Docker's automatic name resolution **inside** `checkers-net`..." (or vice versa)

Reply:
Perhaps:

If running from Docker, you have to pass to the tests values that can be resolved via Docker's automatic name resolution within `checkers-net`: